### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by cben**
Reopening of https://github.com/moolitayer/image-inspector-client/issues/20 in new repo.

cc @enoodle

### CODE REVIEW

File: lib/image-inspector-client/version.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Client for image inspector: https://github.com/openshift/image-inspector
 module ImageInspectorClient
-  VERSION = '1.0.3'
+  VERSION = '1.0.3'.freeze
```
Comment (enoodle) [inline lib/image-inspector-client/version.rb:3]: @cben  Do we intend to bump this in another PR?

Comment (cben) [reply lib/image-inspector-client/version.rb:3]: yes.
Well I opened this PR from Mooli's branch, which felt clever but doesnt allow me to do any changes :-)

**Comment by cben** [comment]
@moolitayer is this the right repo?  Do you have merge rights here?

**Comment by moolitayer** [comment]
Yes and Yes @cben

**Review by moolitayer — APPROVED** [review]
LGTM :+1:

**Comment by cben** [comment]
@moolitayer can we now bump versions and release?
image-inspector itself is still stuck unreleased downstream, but I hope that doesn't prevent the client getting released?

The hard dependency on `recursive-open-struct = 1.0.0` in last released version of this gem (1.0.3) prevents updating manageiq to recent kubeclient versions...

**Comment by moolitayer** [comment]
https://github.com/openshift/image-inspector-client-ruby/pull/4

**Comment by moolitayer** [comment]
2.0.0 released https://rubygems.org/gems/image-inspector-client/versions/2.0.0

**Pull request merged**